jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -520,6 +522,36 @@ public void setScanDepth(int scanDepth)
         _scanDepth = scanDepth;
     }
 
+    /**
+     * Test if scanning should start automatically with Scanner.{@link #start()}
+     *
+     * @return true if scanning should start automatically, false to have scanning is deferred to a later manual call to {@link #startScanning()}
+     */
+    public boolean isAutoStartScanning()
```

Comment:
I don't love this name... but not a blocker for me.   But how about:
```suggestion
    public boolean isScanAtStart()
```